woocommerce/woocommerce
# Update logo icon for Google Listings and Ads in onboarding marketing task

### All Submissions:


### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/35731.

In this PR, we change the logo icon for Google Listings and Ads in the WooCommerce onboarding marketing task (the default tasks with marketplace suggestions turned off in WooCommerce Settings), from the Google Ads icon to Google's "Big G" icon.

The change here is to be consistent with PR https://github.com/woocommerce/woocommerce/pull/34471.

I removed the old `google-listings-and-ads.png` and added `google.svg` file. `google.svg` is a copy of https://github.com/woocommerce/woocommerce/blob/6b3056422eee798f9a1764fc82657d92534e09d2/plugins/woocommerce/assets/images/marketing/google.svg. I checked that there are no more references to `google-listings-and-ads.png`.

<img width="939" alt="image" src="https://user-images.githubusercontent.com/417342/204018556-dd6a2920-f309-454d-a7bd-a37c61ac0a12.png">

### How to test the changes in this Pull Request:

UI test:

1. Make sure marketplace suggestion is turned off in WooCommerce Settings > Advanced > WooCommerce.com: `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com`.
2. Go to WooCommerce Onboarding Marketing task: `/wp-admin/admin.php?page=wc-admin&task=marketing`.
3. The Google Listings and Ads icon should be the "Big G" icon.

Code base check: 

1. Make sure there are no more references to `google-listings-and-ads.png`.

### Other information:


### FOR PR REVIEWER ONLY:
